README.md

```diff
@@ -117,6 +117,12 @@ For example:
 footloose config create --replicas 3 --image quay.io/footloose/fedora29
 ```
 
+Ubuntu images need the `--privileged` flag:
```

Comment:
This is only true for Ubuntu 16.04 it seems. 18.04 works without that flag here. Would you mind giving it a quick tests on your machine and specify 16.04 in that sentence if that's indeed the case?

Thanks!

Reply:
This a need in Ubuntu 16.04. I saw an issue about it, but lost the link since. I'll try anyways. :)

Reply:
My point was, if it's a 16.04 only issue (likely because of the version of systemd then), the sentence needs to say 16.04, not all Ubuntu images :)

Reply:
Right, right. I'm just confirming. :) And yeah the sentence needs to say only 16.04. :)

Reply:
Ubuntu 18.04 gives the following output with systemd-resolved.service errors when not using `--privileged`.

```
systemd 237 running in system mode. (+PAM +AUDIT +SELINUX +IMA +APPARMOR +SMACK 
[...snip...]
[  OK  ] Reached target Swap.
systemd-resolved.service: Failed to reset devices.list: Operation not permitted
Failed to set devices.allow on /docker/b7f77c63c5f6dc0d2c01d2b1aae9711f2ab2be93761359e84c78aab17ce634f5/system.slice/systemd-resolved.service: Operation not permitted
Failed to set devices.allow on /docker/b7f77c63c5f6dc0d2c01d2b1aae9711f2ab2be93761359e84c78aab17ce634f5/system.slice/systemd-resolved.service: Operation not permitted
Failed to set devices.allow on /docker/b7f77c63c5f6dc0d2c01d2b1aae9711f2ab2be93761359e84c78aab17ce634f5/system.slice/systemd-resolved.service: Operation not permitted
Failed to set devices.allow on /docker/b7f77c63c5f6dc0d2c01d2b1aae9711f2ab2be93761359e84c78aab17ce634f5/system.slice/systemd-resolved.service: Operation not permitted
Failed to set devices.allow on /docker/b7f77c63c5f6dc0d2c01d2b1aae9711f2ab2be93761359e84c78aab17ce634f5/system.slice/systemd-resolved.service: Operation not permitted
Failed to set devices.allow on /docker/b7f77c63c5f6dc0d2c01d2b1aae9711f2ab2be93761359e84c78aab17ce634f5/system.slice/systemd-resolved.service: Operation not permitted
Failed to set devices.allow on /docker/b7f77c63c5f6dc0d2c01d2b1aae9711f2ab2be93761359e84c78aab17ce634f5/system.slice/systemd-resolved.service: Operation not permitted
Failed to set devices.allow on /docker/b7f77c63c5f6dc0d2c01d2b1aae9711f2ab2be93761359e84c78aab17ce634f5/system.slice/systemd-resolved.service: Operation not permitted
Failed to set devices.allow on /docker/b7f77c63c5f6dc0d2c01d2b1aae9711f2ab2be93761359e84c78aab17ce634f5/system.slice/systemd-resolved.service: Operation not permitted
Failed to set devices.allow on /docker/b7f77c63c5f6dc0d2c01d2b1aae9711f2ab2be93761359e84c78aab17ce634f5/system.slice/systemd-resolved.service: Operation not permitted
         Starting Network Name Resolution...
[  OK  ] Started Journal Service.
         Starting Flush Journal to Persistent Storage...
[...snip...]
```

Reply:
Ok, thanks a lot for testing. There are some interesting differences we will need to get to the bottom of but, for now, adding the sentence makes sense!